### PR TITLE
[1.12] Bump dcos-signal version

### PR DESCRIPTION
**Pull request by drewvanstone**
## High-level description

Bumps signal version and corrects the licensing socket location

## Corresponding DC/OS tickets (required)

  - [DCOS-55882](https://jira.mesosphere.com/browse/DCOS-55882) 

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [x] Test Results: [link to CI job test results for component]
  - [x] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by drewvanstone** [comment]
@mesosphere-mergebot label Ready For Review

File: CHANGES.md [hunk 1]
```diff
@@ -31,6 +33,8 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 * Updated ref of dvdcli to fix dvdcli package build (DCOS-53581)
 
 * Updated urllib3 version to 1.24.2 due to: https://nvd.nist.gov/vuln/detail/CVE-2019-11324. (DCOS-52210)
+* Updated to [Metronome 0.6.33](https://github.com/dcos/metronome/tree/b8a73dd)
```
Comment (makkes) [inline CHANGES.md]: Does that really belong here?

File: CHANGES.md [hunk 2]
```diff
@@ -31,6 +33,8 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
...
+* Updated to [Marathon 1.8.207](https://github.com/mesosphere/marathon/tree/9f3550487).
```
Comment (makkes) [inline CHANGES.md]: Does this really belong here?

File: CHANGES.md (hunk 1 above)
Comment (drewvanstone) [reply CHANGES.md]: Good catch, just fixed

File: CHANGES.md (hunk 2 above)
Comment (drewvanstone) [reply CHANGES.md]: Fixed!

**Comment by makkes** [comment]
@mesosphere-mergebot sync

**Pull request merged**